data_generator/erc20-gateways/deposit_proof.js

```diff
@@ -0,0 +1,154 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const BN = require('bn.js');
+const rlp = require('rlp');
+const fs = require('fs');
+
+const ERC20Gateway = artifacts.require('ERC20Gateway');
+
+const Web3 = require('web3');
+const { AccountProvider } = require('../../test/test_lib/utils');
+const Utils = require('../../test/test_lib/utils');
+// const web3 = require('../../test/test_lib/web3');
+
+contract('ERC20Gateway::deposit', async (accounts) => {
+  const accountProvider = new AccountProvider(accounts);
+  let erc20Gateway;
+  let erc20Cogateway;
+  let stateRootProvider;
+  let setupParam;
+  let valueToken;
+  let depositor;
+  let param;
+  let outboundChannelIdentifier;
+  let blockNumber;
+  let web3;
+
+  function storagePath(
+    storageIndex,
+    mappings,
+  ) {
+    let path = '';
+
+    if (mappings && mappings.length > 0) {
+      mappings.map((mapping) => {
+        path = `${path}${web3.utils.padLeft(mapping, 64)}`;
+        return path;
+      });
+    }
+
+    path = `${path}${web3.utils.padLeft(storageIndex, 64)}`;
+    path = web3.utils.sha3(path);
+
+    return path;
+  }
+
+  function formatProof(proof) {
+    const formattedProof = proof.map(p => rlp.decode(p));
+    return `0x${rlp.encode(formattedProof).toString('hex')}`;
+  }
+
+
+  beforeEach(async () => {
+    web3 = new Web3('http://localhost:8545');
```

Comment:
pull out constant for end point.

Reply:
File deleted, due to the same reason as above.